File: examples/lh-server/go/16-user-tasks/workflow.go
package main

import "github.com/littlehorse-enterprises/littlehorse/sdk-go/littlehorse"

func wfLogic(wf *littlehorse.WorkflowThread) {
	userId := wf.DeclareStr("user-id").Searchable().Required()
	userTaskDefName := "report-favorite-player"
	wf.AssignUserTask(userTaskDefName, userId, nil)

	wf.Execute("report-favorite-player", userId)
}

File: examples/lh-server/python/16-user-tasks/workflow.py
from littlehorse.workflow import Workflow, WorkflowThread


def get_workflow() -> Workflow:
    def wfLogic(wf: WorkflowThread) -> None:
        user_id = wf.declare_str("user-id").searchable().required()
        favorite_team = wf.declare_str("favorite-team")
        favorite_number = wf.declare_int("favorite-player-number")
        user_task_def_name = "report-favorite-player"
        form_result = wf.assign_user_task(user_task_def_name, user_id)
        favorite_team.assign(form_result.with_json_path("$.favorite-team"))
        favorite_number.assign(form_result.with_json_path("$.favorite-player-number"))
        wf.execute("report-favorite-player", user_id, favorite_team, favorite_number)

    return Workflow("favorite-player-demo", wfLogic)
